SekoiaIO/activity_log/ingest/parser.yml

```
@@ -46,11 +46,9 @@ stages:
           sekoiaio.activity.archive_view.uuid: "{{json_event.message.action.parameters.archive_view.uuid}}"
           sekoiaio.activity.archive.name: "{{json_event.message.action.parameters.archive.name}}"
           sekoiaio.activity.archive.provider: "{{json_event.message.action.parameters.archive.provider}}"
-          sekoiaio.activity.archive.provider: "{{json_event.message.action.parameters.archive.provider}}"
           sekoiaio.activity.archive.short_id: "{{json_event.message.action.parameters.archive.short_id}}"
           sekoiaio.activity.archive.snapshot.uuid: "{{json_event.message.action.parameters.archive.snapshot.uuid}}"
-          sekoiaio.activity.archive.uuid: "{{json_event.message.action.parameters.archive.uuid}}"
-          sekoiaio.activity.archive.uuid: "{{json_event.message.action.results.archive.uuid}}"
+          sekoiaio.activity.archive.uuid: "{{json_event.message.action.parameters.archive.uuid or json_event.message.action.results.archive.uuid}}"
```

A solution can be:
```suggestion
          sekoiaio.activity.archive.uuid: "{{json_event.message..get('action', {}).get('parameters', {}).get('archive', {}).get('uuid') or json_event.message.get('action', {}).get('results', {}).get('archive', {}).get('uuid')}}"
```
Or to add a filter to test the fields.